src/grokcore/catalog/tests/functional/catalog/catalog.py

```diff
@@ -30,6 +30,7 @@
   >>> intids = component.getUtility(IIntIds)
   >>> sm.unregisterUtility(intids, provided=IIntIds)
   True
+  >>> setSite(None)
```

Comment:
It might make for a much smaller diff  if the the functional test layer was tweaked to do this in `testTearDown()`. (There would have to be a new subclass of `ZODBLayer`.)